### PR TITLE
Opta FAQ collection [HC-1523]

### PR DESCRIPTION
**Pull request by gorillagripcore**

### CODE REVIEW

File: content/Hardware Support/Portenta Family/Opta-FAQ.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: "Opta FAQ"
+---
+
+The frequently asked questions covered in this article are focused on providing essential information about The Arduino Opta® and how to operate it with ease.
+
+## FAQ
```
Comment (seaxwi) [inline content/Hardware Support/Portenta Family/Opta-FAQ.md:7]:
```suggestion
---
```

File: content/Hardware Support/Portenta Family/Opta-FAQ.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+---
+title: "Opta FAQ"
+---
+
+The frequently asked questions covered in this article are focused on providing essential information about The Arduino Opta® and how to operate it with ease.
```
Comment (seaxwi) [inline content/Hardware Support/Portenta Family/Opta-FAQ.md:5]:
```suggestion
The frequently asked questions covered in this article are focused on providing essential information about Arduino Opta® and how to operate it with ease.
```

**Review by seaxwi — CHANGES_REQUESTED** [review]

**Review by i-herrera — APPROVED** [review]
Looks good to me!

File: content/Hardware Support/Portenta Family/FAQ-Arduino-Opta.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+Yes, data logging is achievable using a USB-C® memory stick.
+
+**What functions do Opta®'s LEDs have?**
+
+Opta® LEDs can be programmed by the user for any desired signaling purpose. The Wi-Fi variant has one Blue LED in addition to the other two variants. There is no difference in its use and can be programmed for any scope. To use the LED above the reset button to indicate that the device is powered on, the user can use the command `digitalWrite(LED_USER, HIGH)` as the initial command in the `setup()`.
+
+**Are there reliability data available for Opta®?**
+
+Yes, the reliability data is provided under MIL-HDBK-217F standards and component information. Opta® Lite and Opta® RS485 have MTBF (Mean Time Between Failures) of 87 years (766,000 h) at 25 °C. They have a survival probability of 92% after 7 years and 89% after 10 years at 25 C. Additional data for other temperatures can be provided upon request.
```
Comment (seaxwi) [inline content/Hardware Support/Portenta Family/FAQ-Arduino-Opta.md:79]: Can you format this as a list?

**Review by seaxwi — CHANGES_REQUESTED** [review]

**Review by seaxwi — APPROVED** [review]

**Pull request merged**